atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/OracleDdlTable.java

```diff
@@ -139,4 +137,12 @@ public void compactInternally() {
             }
         }
     }
+
+    private String prefixedTableName() {
```

Comment:
Yes, I considered making there be a static utility, but it is nice to have the function be condensed with private state (it makes the actual SQL strings easier to read imo).  But yeah, I'm happy to dedup this (or possibility even do other refactors to dedup this).